TestProjects/ShaderFoundry/Assets/CommonAssets/Editor/EditorTests/BlockLibrary/BlockBuilderUtilities.cs

```diff
@@ -52,18 +61,38 @@ internal static Block.Builder CreateSimplePropertyBlockBuilder(ShaderContainer c
             var blockBuilder = new Block.Builder(container, blockName);
 
             // Build the input type
-            var inputTypeBuilder = new ShaderType.StructBuilder(blockBuilder, "Input");
-            var fieldName = propertyData.FieldName;
-            var inputFieldBuilder = new StructField.Builder(container, fieldName, propertyData.FieldType);
-            if (propertyData.PropertyAttribute != null)
-                MarkAsProperty(container, inputFieldBuilder, propertyData.PropertyAttribute);
+            var inputFields = new List<FieldData>();
+            // TODO @ SHADERS: This should get cleaned up eventually not not have the primary field, but all of the tests need to get updated.
```

Comment:
Nit not not

Reply:
was actually supposed to be "to not", good catch.